### PR TITLE
APEXMALHAR-2304 SQL Support Examples

### PR DESCRIPTION
**Pull request by chinmaykolhatkar**
This is example for newly added SQL support in apex-malhar.

### CODE REVIEW

**Comment by chinmaykolhatkar** [comment]
@tweise Please review and merge.

**Comment by tweise** [comment]
I wonder if it is possible to reduce repetitive boilerplate populateDAG code so that it looks more like run(sql)? Otherwise this looks rather complicated to a user.

File: demos/sql/pom.xml [hunk 1]
```diff
@@ -0,0 +1,223 @@
...
+    </dependency>
+    <dependency>
+      <groupId>org.apache.apex</groupId>
+      <artifactId>malhar-sql</artifactId>
+      <version>${project.version}</version>
+    </dependency>
+
+    <!-- Dependency for malhar-sql... Figure out how can this be avoided -->
+    <dependency>
+      <groupId>org.apache.apex</groupId>
```
Comment (tweise) [inline demos/sql/pom.xml]: This shouldn't be necessary when malhar-sql is properly setup, those should already be transitive dependencies. Except for those cases where they were marked provided.

**Comment by chinmaykolhatkar** [comment]
@tweise The examples mentioned in this are for using the existing APIs of malhar-sql.
Its very much possible to reduce boilerplate code... That's part of APEXMALHAR-2293... This is being worked on now.
Once this is done, one could use only SQL to execute the DAG in following 2 ways:
1. 

public void populateDAG(DAG dag, Configuration conf)
{
  runSQL(dag, conf, <List of sql statements to execute>);
}
1. Input can be a file where only SQL statements can be mentioned to a command line utility and then the cmdline utility will launch the DAG.

File: demos/sql/pom.xml (hunk 1 above)
Comment (chinmaykolhatkar) [reply demos/sql/pom.xml]: Done.

**Comment by chinmaykolhatkar** [comment]
@tweise Can you review and merge?

**Pull request merged**